LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -83,176 +94,347 @@ function OnRadioTowerUpgradeClicked(UIButton Button)
+	}
+
+	// KDM : Rebel job column header
     JobHeaderButton = Spawn(class'UIButton', HeaderPanel);
-    JobHeaderButton.InitButton(, m_strMission, OnJobHeaderButtonClicked);
-    JobHeaderButton.SetResizeToText(false);
-    JobHeaderButton.SetWidth(HeaderPanel.Width * 0.3f);
-    JobHeaderButton.SetPosition(NameHeaderButton.X + NameHeaderButton.Width + 2, 0);
+	JobHeaderButton.bAnimateOnInit = false;
+	JobHeaderButton.bIsNavigable = false;
+	JobHeaderButton.ResizeToText = false;
+    JobHeaderButton.InitButton(, CAPS(m_strMission), OnJobHeaderButtonClicked);
+    JobHeaderButton.SetPosition(NextX, 0);
+	JobHeaderButton.SetSize(AvailableSpace * m_JobHeaderPct, 30);
+	JobHeaderButton.SetStyle(eUIButtonStyle_NONE, m_HeaderFontSize);
     JobHeaderButton.SetWarning(true);
+	JobHeaderButton.ProcessMouseEvents(OnJobHeaderMouseEvent);
 
     NextY += 32 + 5;
- 
+
+	// KDM : List container which will hold rows of rebels
 	List = Spawn(class'UIList', MainPanel);
+	List.bAnimateOnInit = false;
 	List.bIsNavigable = true;
-	List.InitList(, 15, NextY, ListBG.Width - 50, panelH - NextY - 5);
-	List.bStickyHighlight = false; 
+	List.bStickyHighlight = false;
+	List.InitList(, m_BorderPadding, NextY, HeaderPanel.Width, panelH - NextY - m_BorderPadding);
 
-	//check to see if we are in geoscape
-	if (`HQGAME == none || `HQPRES == none || `HQPRES.StrategyMap2D == none)
+	// KDM : As per Long War 2, radio relays can only be built via the strategy map, not via the Avenger.
+	if (!class'Utilities_LW'.static.IsOnStrategyMap())
 	{
 	}
 	else
 	{
 		RadioTowerUpgradeButton = Spawn(class'UIButton', MainPanel);
+		RadioTowerUpgradeButton.bAnimateOnInit = false;
+		RadioTowerUpgradeButton.bIsNavigable = false;
 		RadioTowerUpgradeButton.InitButton(, m_strBuildRadioRelay, OnRadioTowerUpgradeClicked);
-		RadioTowerUpgradeButton.AnchorTopLeft();
-		RadioTowerUpgradeButton.OriginTopRight();
-		RadioTowerUpgradeButton.SetPosition(1430, 168);
-		//RadioTowerUpgradeButton.Hide();
+		// KDM : I did a lot of testing, and this button seems to anchor to the screen rather than MainPanel.
+		// Therefore, to make things easier, I chose manual positioning rather than positioning through the use of an anchor and origin.
+		RadioTowerUpgradeButton.SetPosition(811, 21);
+		RadioTowerUpgradeButton.SetHeight(30);
+		RadioTowerUpgradeButton.SetFontSize(m_HeaderFontSize);
 	}
-    // Redirect all mouse events for the background to the list. Ensures all mouse
-    // wheel events get processed by the list instead of consumed by the background
-    // when the cursor falls "between" list items.
-    ListBG.ProcessMouseEvents(List.OnChildMouseEvent);
 
-    InitJobNameCache();
+    // LWS : Redirect all background mouse events to the list so mouse wheel scrolling doesn't get lost when the mouse is positioned between list items.
+	ListBG.ProcessMouseEvents(List.OnChildMouseEvent);
 
+    InitJobNameCache();
     RefreshNavHelp();
     GetData();
     RefreshData();
-    Show();
+   
```

Comment:
Going back to my comment on indent, since the above three lines are within a block of other changed code, I would change their indent despite them not being otherwise different from the original code.